docs/reference/eql/functions.asciidoc

```diff
@@ -276,4 +277,82 @@ function returns the remaining string.
 Positions are zero-indexed. Negative offsets are supported.
 
 *Returns:* string
+====
+
+[discrete]
+[[eql-fn-wildcard]]
+=== `wildcard`
+Returns `true` if a source string matches one or more provided wildcard
+expressions. Matching is case sensitive.
```

Comment:
> Matching is case sensitive.
This is likely to change with #54411. However, I'm leaving this in until those changes are implemented.